include/constants/items.h

```diff
@@ -35,8 +35,8 @@
 // Note: If moving ball IDs around, updating FIRST_BALL/LAST_BALL is not sufficient
```

Comment:
This note needs to be removed or changed.

Reply:
I removed those defines since we can just use `POKEBALL_COUNT` now